### PR TITLE
complete ticket 4 & 5

### PR DESCRIPTION
**Pull request by moskgit**
Hi, I haven't had chance to create a pull request as it was after 4:30pm. I worked for two tickets and can't differentiate now! Sorry for that. I added a lot of images and folders to the repo, that's why it may seem a lot. Please check it for me. Thank you for your help.

### CODE REVIEW

**Review by yalicee — CHANGES_REQUESTED** [review]
Please don't merge different tickets into 1 just because you missed the 4:30 deadline. I will allow you to have tickets 4 and 5 this time but you seem to have code for further tickets which I would like you to remove

**Comment by moskgit** [comment]
Hi Alice, 
Thank you for your understanding. In order to follow the drafted plan, I had to layout the components that's all, which may seem like extras, but I really had no choice if I were to follow the plan. I haven't done anything for any further ticket just yet. Yet, you know it better, so if you can point out what would you like me to remove, i'll be thankful. 
Thank you for your patience.

**Comment by yalicee** [comment]
There is code for further tickets as I can see code for api requests when there shouldn't be etc.

File: src/App.css [hunk 1]
```diff
@@ -1,63 +1,198 @@
...
-}
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+/* The dropdown container */
+.categWrapper {
+  margin: auto;
+  height: auto;
+  display: flex;
+  padding-bottom: 0%;
+  align-content: baseline;
```
Comment (roisinoreilly) [inline src/App.css:111]: nice consideration for styling so far!

File: src/Components/Main/Articles/SingleArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import ArticlesList from "./ArticlesList";
+
+
+export default function SingleArticle () {
+
+    const [articlesData, setArticlesData] = useState([]);
+    const {article_id} = useParams();
+
+    useEffect(() => {
+        getArticlesById(article_id)
```
Comment (roisinoreilly) [inline src/Components/Main/Articles/SingleArticle.jsx:13]: nicely done

File: src/Components/Main/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        .then(({articles}) => {
+            setArticlesData(articles);
+          })
+        .catch(err => console.log(err))
+        .finally(()=> {
+            setIsLoading(false);
+        })
+      },[topicsData, articlesData]);
+  
+      if(isLoading) return <h2 className='Loading'>Loading...</h2>
```
Comment (roisinoreilly) [inline src/Components/Main/Main.jsx:29]: this could be extracted into it's own component so it can be reusable

**Review by roisinoreilly — APPROVED** [review]
ticket 4 - view all articles
- the timestamp isn't very user friendly, can you try manipulating the data to format it into something human readable, or look into some packages that can do it for you
- you've clearly considered the user experience and future features you plan to implement which is good, undoubtedly the final product will be an excellent and well-fleshed out app
- I don't think the article id needs to be displayed to the user, this is more for development use

ticket 5 - view single articles
- good job extracting your code into good size components
- a loading page would be good to have, while the individual article comes back from the API the user just sees a blank screen
- the same as above regarding the timestamp and article id

there is code here not just for ticket 4 and 5 but for other future tickets like sorting, comments, and login that you don't need, you need to be careful and stick strictly to only writing code for the ticket you're working on, not just for this but for future practice. Furthermore, it's common practice in development to have to wait a lot longer for pull requests to be approved and waiting 2 or 3 weeks is normal, it's okay to follow them up to check we received them since we won't take this long, but asking senior devs to hurry up won't be met with a good reception

PR approved, just don't send two tickets in one request again please!

**Pull request merged**